ports/mimxrt/modules/_boot.py

```diff
@@ -3,13 +3,32 @@
 # Note: the flash requires the programming size to be aligned to 256 bytes.
 
 import os
+import sys
 import mimxrt
+from machine import Pin
 
 bdev = mimxrt.Flash()
-
 try:
     vfs = os.VfsLfs2(bdev, progsize=256)
 except:
     os.VfsLfs2.mkfs(bdev, progsize=256)
     vfs = os.VfsLfs2(bdev, progsize=256)
-os.mount(vfs, "/")
+os.mount(vfs, "/flash")
+os.chdir("/flash")
+sys.path.append("/flash")
+
+# do not mount the SD card if SKIPSD exists.
+try:
+    os.stat("SKIPSD")
+except:
+    try:
+        from machine import SDCard
+
+        sdcard = SDCard(1)
+        if sdcard:
```

Comment:
this `if` shouldn't be needed as `sdcard` will always be true

Reply:
`sdcard` can be `None`if no card is detected. So I would like to keep the `if`statement.